Lib/test/support/strace_helper.py

```diff
@@ -72,6 +72,27 @@ def sections(self):
         return sections
 
 
+def filter_memory(syscalls):
+    """Filter out memory allocation calls from File I/O calls.
+
+    Some calls (mmap, munmap, etc) can be used on files or to just get a block
+    of memory. Use this function to filter out the memory related calls from
+    other calls."""
+
+    def _filter(call):
+        # mmap can operate on a fd or "MAP_ANON" which gives a block of memory.
+        # Ignore the "MAP_ANON" ones.
+        if call.syscall == "mmap" and "MAP_ANON" in call.args[3]:
+            return False
```

Comment:
Would it be possible to return True if the syscall should be filtered, and False otherwise? In short, replace `if _filter(call)` with `if not _filter(call)`.